docs/porting.md

```diff
@@ -40,6 +40,46 @@ free-threaded build. Since the changes required in native extensions are more
 substantial, we have split off the guide for porting extension modules into
 [a subsequent section](porting-extensions.md).
 
+### Define and document thread safety guarantees
+
+!!! note
+
+    This advice is primarily targeted at library authors or anyone who maintains Python code that others might use in their own application or library.
+
+Broadly, we think a Python package can indicate support for the free-threaded
+build at one of four levels:
+
+- Not supported at all.
+- Supported for experimental use, multithreaded use may lead to issues
```

Comment:
This is phrased in a significantly more negative way than https://github.com/pypa/trove-classifiers/issues/177#issuecomment-2351953921. I'd use that instead: _"Free threaded usage is supported, but documentation of constraints and limitations may be incomplete."_